crates/python/src/qvm/api.rs

```diff
@@ -123,7 +123,7 @@ impl PyMultishotRequest {
 
     #[setter]
     pub fn set_trials(&mut self, trials: u16) -> PyResult<()> {
-        // `NonZeroU16` doesn't implement `PyClass`, so `pyo3` doesn't allow it to be used
+        // `NonZeroU32` doesn't implement `PyClass`, so `pyo3` doesn't allow it to be used
```

Comment:
Looks like `u16` is being used in this area (`fn trials` and `fn set_trials`) when it should be `u32`

(Commenting here because GH does not let me comment on non-diff lines)

Reply:
This also happens below in functions with the same names.